File: tests/fixtures/selenium-chaos-suite-csharp/Utilities/AssertionHelper.cs
using NUnit.Framework;

namespace SeleniumChaosSuite.Utilities;

public static class AssertionHelper
{
    public static void ShouldContain(string actual, string expected, string note = "")
    {
        Assert.That(actual, Does.Contain(expected), note);
    }

    public static void ShouldEqual(object? actual, object? expected, string note = "")
    {
        Assert.That(actual, Is.EqualTo(expected), note);
    }

    public static void ShouldBeGreaterThan(double actual, double expected, string note = "")
    {
        Assert.That(actual, Is.GreaterThan(expected), note);
    }

    public static void ShouldBeTruthy(bool actual, string note = "")
    {
        Assert.That(actual, Is.True, note);
    }
}

File: tests/fixtures/selenium-chaos-suite/src/core/AssertionHelper.ts
import { expect } from "chai";

export class AssertionHelper {
  static shouldContain(actual: string, expected: string, note = "") {
    expect(actual, note).to.contain(expected);
  }

  static shouldEqual(actual: any, expected: any, note = "") {
    expect(actual, note).to.equal(expected);
  }

  static shouldBeGreaterThan(actual: number, expected: number, note = "") {
    expect(actual, note).to.be.greaterThan(expected);
  }

  static shouldBeTruthy(actual: any, note = "") {
    expect(actual, note).to.equal(true);
  }
}
